src/Tests/Microsoft.Diagnostics.Monitoring.UnitTestApp/Scenarios/ParameterCapturingScenario.cs

```diff
@@ -6,91 +6,62 @@
-            scenarioCommand.Subcommands.Add(expectLogStatementsCommand);
-            scenarioCommand.Subcommands.Add(doNotExpectLogStatementsCommand);
             scenarioCommand.Subcommands.Add(aspNetAppCommand);
             scenarioCommand.Subcommands.Add(nonAspNetAppCommand);
+            scenarioCommand.Subcommands.Add(aspNetAppWithSampleCommand);
 
             return scenarioCommand;
         }
 
-        public static Task<int> ExpectLogStatementAsync(ParseResult result, CancellationToken token)
-        {
-            return LogStatementCoreAsync(result, expectLogs: true, token);
-        }
-
-        public static Task<int> DoNotExpectLogStatementAsync(ParseResult result, CancellationToken token)
-        {
-            return LogStatementCoreAsync(result, expectLogs: false, token);
-        }
-
-        private static Task<int> LogStatementCoreAsync(ParseResult result, bool expectLogs, CancellationToken token)
+        public static Task<int> AspNetAppAsync(ParseResult result, CancellationToken token)
         {
-            LogRecord logRecord = new();
-
             return ScenarioHelpers.RunWebScenarioAsync<Startup>(
                 func: async logger =>
                 {
-                    await ScenarioHelpers.WaitForCommandAsync(TestAppScenarios.ParameterCapturing.Commands.Validate, logger);
-
-                    SampleMethods.StaticTestMethodSignatures.NoArgs();
-
-                    bool didFindLogs = logRecord.Events.Where(e => e.Category == typeof(DotnetMonitor.ParameterCapture.UserCode).FullName).Any();
-                    Assert.Equal(expectLogs, didFindLogs);
+                    await ScenarioHelpers.WaitForCommandAsync(TestAppScenarios.ParameterCapturing.Commands.Continue, logger);
                     return 0;
-
-                },
-                token,
-                configureServices: (services) =>
-                {
-                    services.AddLogging(builder =>
-                    {
-                        builder.AddProvider(new TestLoggerProvider(logRecord));
-                    });
-                });
+                }, token);
         }
 
-        public static Task<int> AspNetAppAsync(ParseResult result, CancellationToken token)
+        public static Task<int> NonAspNetAppAsync(ParseResult result, CancellationToken token)
         {
-            return ScenarioHelpers.RunWebScenarioAsync<Startup>(
+            return ScenarioHelpers.RunScenarioAsync(
                 func: async logger =>
                 {
                     await ScenarioHelpers.WaitForCommandAsync(TestAppScenarios.ParameterCapturing.Commands.Continue, logger);
                     return 0;
                 }, token);
         }
 
-        public static Task<int> NonAspNetAppAsync(ParseResult result, CancellationToken token)
+        public static Task<int> AspNetAppWithSampleMethodAsync(ParseResult result, CancellationToken token)
```

Comment:
Do we really need a separate scenario for this? Could you just combine this with AspNetAppAsync and then have the existing test code that using AspNetAppAsync just issue the Continue command without checking what methods were captured?